### PR TITLE
news/blog: Managing NFS exports with Dashboard

### PR DESCRIPTION
**Pull request by alfonsomthd**
Signed-off-by: alfonsomthd <almartin@redhat.com>

Preview:
https://managing-nfs-exports-with-dashboard.ceph.io/en/news/blog/2021/managing-nfs-exports-with-dashboard/

### CODE REVIEW

**Review by nizamial09 — APPROVED** [review]

**Review by avanthakkar — APPROVED** [review]

**Review by Thingee — APPROVED** [review]

File: src/en/news/blog/2021/managing-nfs-exports-with-dashboard/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+So let's go to **Cluster > Services** and create an NFS service:
+
+![Create NFS service](images/create-nfs-service.png)
+
+The NFS service is running:
+
+![NFS service details](images/nfs-service-details.png)
+
+We return to **Dashboard > NFS > Create Export** and create an export with CephFS as the Storage Backend:
```
Comment (epuertat) [inline src/en/news/blog/2021/managing-nfs-exports-with-dashboard/index.md:44]: I'd just add level 3 headings for "NFS over CephFS" and "NFS over RGW".

**Review by epuertat — APPROVED** [review]
LGTM! Thanks @alfonsomthd !

File: src/en/news/blog/2021/managing-nfs-exports-with-dashboard/index.md (hunk 1 above)
Comment (alfonsomthd) [reply src/en/news/blog/2021/managing-nfs-exports-with-dashboard/index.md:50]: Done! I also added `NFS service deployment` header for clarification.

**Review by pereman2 — APPROVED** [review]

**Comment by alfonsomthd** [comment]
@Thingee I think this is ready to be merged.

**Comment by alfonsomthd** [comment]
> @Thingee I think this is ready to be merged.

@Thingee ping.

**Comment by Thingee** [comment]
Hi everyone, sorry for the delays as I just returned from PTO. Thank you Alfonso!

**Pull request merged**